test/helpers/assignments_helper_test.rb

```diff
@@ -1,6 +1,6 @@
 class AssignmentsHelperTest < ActionView::TestCase
   test "hashes should be unique" do
-  	max_value = 26**4-1 # 4 letters, 26 options, 0 indexed
+  	max_value = 20**4-1 # 4 letters, 26 options, 0 indexed

```

Comment:
The comment here is not updated--it should be `4 letters, 20 options, 0 indexed`